vllm/attention/backends/flashinfer.py

```diff
@@ -217,10 +247,14 @@ def forward(
                 self.kv_cache_dtype,
             )
 
+        query = query.contiguous(
```

Comment:
I think we discussed this before, but what's the overhead of this call?

Reply:
For llama7b on A100, the shape of query is [256, 12, 64], and this line takes ~0.037ms.